extension/EdFi.Ods.Extensions.LMSX/LMS-Harmonizer/1080-lms-assignment.sql

```diff
@@ -66,7 +65,7 @@ BEGIN
 		,[SchoolYear]
 		,[SchoolId]
 		,[Namespace])
-	select
+	SELECT
```

Comment:
All caps isn't necessary. However, the use of lowercase in just a few places was inconsistent with the rest of the file. So I updated it.